### PR TITLE
Group membership telemetry updates

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost-server/pull/20437
- Updated Group Discovery Diagnostics (Enterprise Edition Only) description to include unique users in groups (custom & AD/LDAP)

### CODE REVIEW

File: source/manage/telemetry.rst [hunk 1]
```diff
@@ -138,7 +138,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Commercial License Diagnostics (Enterprise Edition Only):* Uploaded an Enterprise license key to the server
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system. Scheme ID; Team Admin permissions; Team user permissions; Channel Admin permissions; Channel user permissions; Number of teams the scheme is associated with
-  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```
Comment (mkraft) [inline source/manage/telemetry.rst:141]: Just a reminder that Custom Groups applies to `Professional` and `Enterprise` licenses.

Comment (cwarnermm) [reply source/manage/telemetry.rst:141]: Great reminder, @mkraft! Thank you! I'll get this updated.

Comment (cwarnermm) [inline source/manage/telemetry.rst:141]:
```suggestion
  - *Group Discovery Diagnostics:* Provides information related to AD/LDAP (Enterprise Edition Only) and custom groups (Enterprise and Professional Edition only), including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```
```suggestion
  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```

**Review by mkraft — APPROVED** [review]

File: source/manage/telemetry.rst [hunk 2]
```diff
@@ -138,7 +138,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
...
   - *Mobile Performance Diagnostics:* Load times for starting the app, switching channels, and switching teams
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system. Scheme ID; Team Admin permissions; Team user permissions; Channel Admin permissions; Channel user permissions; Number of teams the scheme is associated with
-  - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP and custom groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
+  - *Group Discovery Diagnostics:* Provides information related to AD/LDAP (Enterprise Edition Only) and custom groups (Enterprise and Professional Edition only), including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```
Comment (justinegeffen) [inline source/manage/telemetry.rst:141]:
```suggestion
  - *Group Discovery Diagnostics:* Provides information related to AD/LDAP (Enterprise Edition only) and custom groups (Enterprise and Professional Edition only), including number of unique users in groups, number of groups synchronized to Mattermost, teams and channels associated to groups, teams and channels synchronized with groups, and number of group members.
```

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**